docs/PRODUCTION.md

```diff
@@ -0,0 +1,364 @@
+# Production configuration
+
+This document describes the recommended setup for an non-containerized Atlas server.
+
+## System requirements
+
+| Dependency | Version | Notes |
+| --- | --- | --- |
+| go | 1.19+ | |
+| gcc | | other C compilers will work fine |
+| systemd | 250+ | 250 is needed for credentials to work <br/> 240 is needed for data directory stuff |
+| logrotate | 3.18.0+ | older versions may also work |
+| zstd | | optional, for logrotate compression |
+| sqlite3 | 3.37+ | 3.37 is needed for strict tables |
+
+## Installation
+
+1. Install Atlas.
+
+    ```bash
+    # option 1: latest version
+    sudo GOBIN=/usr/local/bin go install -v -trimpath github.com/r2northstar/atlas/cmd/...@main
+    ```
+
+    ```bash
+    # option 2: from local git clone
+    git clone https://github.com/r2northstar/atlas
+    cd atlas
+    sudo GOBIN=/usr/local/bin go install -v -trimpath github.com/r2northstar/atlas/cmd/...
```

Comment:
out of curiosity, what's with the triple `...`? ^^

Reply:
Think of it as a `**` glob, but a Go thing and for matching folders (i.e., packages) only.